src/main/java/org/fenixedu/ulisboa/specifications/service/evaluation/MarkSheetImportExportService.java

```diff
@@ -23,19 +20,19 @@
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Maps;
 
+import static com.qubit.qubEdu.module.base.util.XLSxUtil.*;
+
 //TODO: improve errors
 public class MarkSheetImportExportService {
```

Comment:
@joaofreitas-qubit was talking with with @sanavarali and this can be deleted, along with the two controllers that are using it (and were changed in this PR as well). So this PR will start having everything deleted, excepted the `AnyCurricularCourseExceptionsConfigurationController.java` that will still be changed